deploy/apigee/policies/rewrite-quota-exceeded-message.template.xml

```diff
@@ -0,0 +1,7 @@
+<AssignMessage async="false" continueOnError="false" enabled="true"
+name="rewrite-quota-exceeded-message">
+  <DisplayName>rewrite-quota-exceeded-message</DisplayName>
+  <Set>
+    <Payload contentType="application/json">{"message": "Quota limit exceeded. If you are using a trial API key, please register at REPLACE_WITH_API_PORTAL_URL to obtain a different key."}</Payload>
```

Comment:
How hard would it be to only display this if they're actually using the trial key?

Reply:
This actually is currently only displayed if they're using the trial key, since the quota policy is only applied to requests that use that particular key.